server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileBackuper.java

```diff
@@ -34,9 +34,14 @@
   void backup(DbSession dbSession, QualityProfileDto profile, Writer backupWriter);
 
   /**
-   * Restore a profile backup in the specified organization. The parameter {@code overriddenProfileName}
-   * is the name of the profile to be used. If null then name is loaded from backup.
+   * Restore backup on a profile in the specified organization. The parameter {@code overriddenProfileName}
+   * is the name of the profile to be used. If null then the name is loaded from backup.
```

Comment:
:mortar_board: I would even say "If **the parameter** is null, then the name is loaded from **the** backup".